wallet/src/main/java/com/android/identity_credential/wallet/ui/destination/provisioncredential/EvidenceRequest.kt

```diff
@@ -598,4 +606,156 @@ fun <ResultT> EvidenceRequestIcaoView(
             }
         }
     }
-}
\ No newline at end of file
+}
+
+@Composable
+fun EvidenceRequestSelfieVideoView(
+    evidenceRequest: EvidenceRequestSelfieVideo,
+    provisioningViewModel: ProvisioningViewModel,
+    permissionTracker: PermissionTracker,
+    walletServerProvider: WalletServerProvider,
+    documentStore: DocumentStore
+) {
+    if (evidenceRequest.poseSequence.isEmpty()) {
+        throw IllegalArgumentException("Pose sequence must not be empty")
```

Comment:
nit: don't mind me, the annoying punctuation guy, but for the sake of consistency with other error messages maybe include here.

Reply:
Done.